rif.config.js

```diff
@@ -1,18 +1,18 @@
 module.exports = {
```

Comment:
I think that this file should have all the mainnet addresses.

And if the user changes the network to RSK Testnet then we need to change the addresses.

Maybe the object should have mainnet and testnet keys.

Btw, this is not strictly related with this pr, but what happens if a user, connected to ethereu, try to use the "my domains" feature?

Reply:
We have to control that, probably or hide the menu or discuss about if we are going to provide support for multiple chains outside the RSK ecosystem.

Reply:
Yes, also, there is no sense in all the rif operations for other chains.